irssi/content.md

```diff
@@ -18,26 +18,22 @@ On a Linux system, build and launch a container named `my-running-irssi` like th
 
 ```console
 $ docker run -it --name my-running-irssi -e TERM -u $(id -u):$(id -g) \
+    --log-driver=none \
     -v $HOME/.irssi:/home/user/.irssi:ro \
     -v /etc/localtime:/etc/localtime:ro \
     irssi
 ```
 
+We specify ``--log-driver=none`` to avoid storing useless interactive terminal data.
+
 On a Mac OS X system, run the same image using:
 
 ```console
 $ docker run -it --name my-running-irssi -e TERM -u $(id -u):$(id -g) \
+    --log-driver=none \
     -v $HOME/.irssi:/home/user/.irssi:ro \
     irssi
 ```
 
 You omit `/etc/localtime` on Mac OS X because `boot2docker` doesn't use this file.
 
-Of course, you can name your image anything you like. In Docker 1.5 you can also use the `--read-only` mount flag. For example, on Linux:
-
-```console
-$ docker run -it --name my-running-irssi -e TERM -u $(id -u):$(id -g) \
-    --read-only -v $HOME/.irssi:/home/user/.irssi \
-    -v /etc/localtime:/etc/localtime \
-    irssi
-```
```

Comment:
I think the idea was to point out that `~/.irssi` is the _only_ place irssi tries to mutate by default.